thirdeye/thirdeye-frontend/tests/integration/pods/components/anomaly-id/component-test.js

```diff
@@ -0,0 +1,50 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('anomaly-id', 'Integration | Component | anomaly id', {
+  integration: true
+});
+
+const idSelector = '.anomaly-id__id';
+const nameSelector = '.anomaly-id__name';
+const currentValueSelector = '.anomaly-id__total';
+const changeRateSelector = '.anomaly-id__percent';
+const anomalyId = 20475;
+const anomalyCurrentValue = 10;
+const anomalyBaselineValue = 2;
+const anomaly = Ember.Object.create({
+  anomalyIds: anomalyId,
+  current: anomalyCurrentValue,
+  baseline: anomalyBaselineValue,
+  anomalyFunctionName: 'some-function-name'
+});
+const calculateChangeRate = (current, baseline) => {
+  return (baseline === 0) ? 0 : ((current - baseline) / baseline * 100).toFixed(2);
+};
+
+// Test for proper rendering of all Anomaly ID Block data
+test('Anomaly ID block: all meta properties render', function(assert) {
+  let changeRate = calculateChangeRate(anomaly.current, anomaly.baseline);
+
+  this.set('anomaly', anomaly);
+  this.set('anomalyChangeRate', changeRate);
+  this.render(hbs`{{anomaly-id anomaly=anomaly}}`);
+
+  assert.equal(this.$(idSelector).text().trim(), anomalyId);
```

Comment:
nit: can we add messages to these assertions. It'll make our life easier if one does fail. 
http://api.qunitjs.com/assert/equal